src/splitInteger.test.js

```diff
@@ -4,18 +4,18 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+expect(splitInteger(20, 5)).toEqual([4, 4, 4, 4, 4])
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+expect(splitInteger(3, 3)).toEqual([1, 1, 1])
```

Comment:
The test case description suggests splitting the number 3 into 1 part, but the test is actually splitting 3 into 3 parts. The expected output `[1, 1, 1]` is correct for splitting 3 into 3 parts, but the test description should be updated to reflect this accurately.